### PR TITLE
Manage account first updates 

### PR DESCRIPTION
**Pull request by stacykor**
## Purpose of this pull request

This pull request (PR) is a draft for CP and team. The account page is going to move account cancellation options to the top, ditch the donut, and in general make the toggles easier. Promotional credits will go down to the bottom. This built locally. 

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by cpawar29 — APPROVED** [review]

File: docs/manage/manage-subscription/cloud-flex-credits-accounts.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ description: View information on Sumo Logic Cloud Flex Credits account and intui
...
+* **Free** accounts give you access to most Sumo Logic features, with a credit allocation of 20 credits a day. Credits can be used for log, metrics, and traces, in any combination that meets your needs. The retention period for logs is 7 days. Free accounts allow three users. Free account holders can [upgrade](upgrade-cloud-flex-account.md) to Sumo Logic paid service agreements in the Sumo Logic UI. 
+* **Trial** accounts allow you to try all of Sumo Logic's advanced features to understand how Sumo Logic will fit within your organization before you buy. It includes a credit allocation to support a daily data volume limit of 1 GB per day, with 20 users and 30 days of data retention. If you use up the credits allocated for the trial period before the period ends, Sumo Logic’s [standard throttling mechanism](../ingestion-volume/log-ingestion.md) will be applied to your log ingest, For information on Trial account features, see [Cloud Flex accounts](#cloud-flex-credits---account-types).
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/cloud-flex-credits-accounts.md:17]:
```suggestion
* **Trial** accounts allow you to try all of Sumo Logic's advanced features to understand how Sumo Logic will fit within your organization before you buy. It includes a credit allocation to support a daily data volume limit of 1 GB per day, with 20 users and 30 days of data retention. If you use up the credits allocated for the trial period before the period ends, Sumo Logic’s [standard throttling mechanism](../ingestion-volume/log-ingestion.md) will be applied to your log ingest, For information on Trial account features, see [Cloud Flex accounts](docs/manage/manage-subscription/cloud-flex-accounts/).  
```

File: docs/manage/manage-subscription/cloud-flex-credits-accounts.md [hunk 2]
```diff
@@ -61,40 +61,49 @@ The following table provides a summary list of key features by Sumo Logic Cloud
...
-* **Usage (Sumo Credits).** Displays the usage of the Sumo Credits you purchased for the contract period.
-* **Contract Capacity Utilization.** Displays the total amount of Sumo Credits used since the beginning of the contract period.
-* **Forecast.** Displays the anticipated end date of the current contract period, along with the expected Sumo Credits usage by the end of the contract period based on the current consumption rate.
+### Usage Forecast
 
-### Usage 
+The usage forecast gives you an opportunity to see the potential end of the current credtits and an estimate on when those credits will end. 
+
+* **Upgrade Plan** Link to the upgrade account page, where you can message your request for an account upgrade to your account manager.
+* **Forecasted use** Predicts your credit usage by end of the contract period.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/cloud-flex-credits-accounts.md:100]:
```suggestion
* **Forecasted use**. Predicts your credit usage by end of the contract period. 
```

File: docs/manage/manage-subscription/cloud-flex-credits-accounts.md [hunk 3]
```diff
@@ -190,35 +191,17 @@ To drill into usage data, do the following:
...
-    ![CFCredits_Usage_Details_drilldown_hover.png](/img/subscriptions/CFCredits_Usage_Details_drilldown_hover.png)
-
-1. To drill down further, repeat step 1 and 2 as needed.
-1. To return to the original data display, click the Reset icon.
-
-    ![CFCredits_Usage_Details_drilldown_reset.png](/img/subscriptions/CFCredits_Usage_Details_drilldown_reset.png)
-
 ### Monitoring Promotional Credit usage
 
 You can filter for a focused credit component to assess your Promotional Credit usage, in the same way you [drill into usage data](/docs/manage/manage-subscription/cloud-flex-credits-accounts/#monitoring-promotional-credit-usage) for paid credits.
```
Comment (jpipkin1) [inline docs/manage/manage-subscription/cloud-flex-credits-accounts.md:198]:
```suggestion
You can filter for a focused credit component to assess your Promotional Credit usage, in the same way you [drill into usage data](#drilling-into-usage-data) for paid credits.
```

File: docs/manage/manage-subscription/cloud-flex-credits-accounts.md [hunk 4]
```diff
@@ -115,11 +123,14 @@ To view the Account page, do the following:
...
     :::
 
-    The following visual indicators apply:
+   Be aware of the following features:
 
-    * Red text indicates an overage of your account limit.
+    * **Details of Credit Usage** allow you to review your consumption by usage categories and credit types.
+    * View consumption by day, week, or month. 
+    * View consumption by time period. 
+    * Download a
```
[the remark this answers is not in the document]
Comment (stacykor) [reply docs/manage/manage-subscription/cloud-flex-credits-accounts.md]: This is in the older draft.

**Review by jpipkin1 — APPROVED** [review]
Looks good, @stacykor.

**Comment by stacykor** [comment]
@cpawar29 are we good to release?

**Pull request merged**